website/content/docs/commands/operator/raft.mdx

```diff
@@ -325,8 +325,10 @@ Flags applicable to this command are the following:
   go without leader contact before being considered unhealthy. Defaults to `10s`.
 
 - `dead-server-last-contact-threshold` `(string)` - Limit on the amount of time
-  a server can go without leader contact before being considered failed. This
-  takes effect only when `cleanup_dead_servers` is set. Defaults to `24h`.
+  a server can go without leader contact before being considered failed.
+  This takes effect only when `cleanup_dead_servers` is set as `true`. Defaults to `24h`.
+  
+  -> **Note:** A failed server cannot rejoin the cluster without being reinitialized.  
```

Comment:
```suggestion
  -> **Note:** A failed server that autopilot has removed from the raft configuration cannot rejoin the cluster without being reinitialized.  
```

This phrasing is a little ambiguous.  A failed server (one exceeding the this threshold) hasn't left the cluster yet.  Based on min_quorum vault we may keep a failed server in the cluster.  It's only once Vault has actually removed a failed server from the raft configuration that this note would apply.